cfy_manager/components/globals.py

```diff
@@ -116,11 +116,12 @@ def _apply_forced_settings():
         config[POSTGRESQL_CLIENT][SSL_ENABLED] = True
 
 
-def set_globals(only_install=False):
+def set_globals(only_install=False, only_constants=False):
```

Comment:
Maybe make this be include_constants=True so that we don't get the negative check later?

Reply:
Actually, ignore this, that'd be the wrong name.